failed to make cross in k8s when go version is 1.13

<!-- This form is for bug reports and feature requests! -->

**Is this a BUG REPORT or FEATURE REQUEST?**:


/kind bug

**The binaries affected**:

- [ ] openstack-cloud-controller-manager
- [ ] cinder-csi-plugin
- [ ] k8s-keystone-auth
- [ ] client-keystone-auth
- [ ] octavia-ingress-controller
- [ ] manila-csi-plugin
- [ ] manila-provisioner
- [ ] magnum-auto-healer
- [ ] barbican-kms-plugin

**What happened**:
After "make build" in cloud provider openstack,  follow the instruction of getting-started to "make cross" in k8s.  Then, there is an error raised:

~/go/src/k8s.io/kubernetes$ make cross

Detected go version: go version go1.13 linux/amd64.
Kubernetes requires go1.13.4 or greater.
Please install go1.13.4 or later.

!!! [0310 03:30:18] Call tree:
!!! [0310 03:30:18]  1: hack/run-in-gopath.sh:31 kube::golang::setup_env(...)

Detected go version: go version go1.13 linux/amd64.
Kubernetes requires go1.13.4 or greater.
Please install go1.13.4 or later.

!!! [0310 03:30:18] Call tree:
!!! [0310 03:30:18]  1: ./hack/run-in-gopath.sh:31 kube::golang::setup_env(...)
Makefile.generated_files:535: recipe for target 'gen_bindata' failed
make[2]: *** [gen_bindata] Error 1
Makefile:544: recipe for target 'generated_files' failed
make[1]: *** [generated_files] Error 2
Makefile:505: recipe for target 'cross' failed
make: *** [cross] Error 1


**What you expected to happen**:
pass the "make cross" without error.

**How to reproduce it**:
Just make cross in k8s.

**Anything else we need to know?**:


**Environment**: ubuntu 18.04.3 LTS
- openstack-cloud-controller-manager version: latest
- OpenStack version: master
- Others: none

--- a/docs/getting-started-provider-dev.md
+++ b/docs/getting-started-provider-dev.md
@@ -120,19 +120,12 @@ sudo yum install -y -q git gcc etcd
 You will also need a recent version of Go and set your environment variables.
 
 ```
-GO_VERSION=1.13
+GO_VERSION=1.13.4
 GO_ARCH=linux-amd64
 curl -o go.tgz https://dl.google.com/go/go${GO_VERSION}.${GO_ARCH}.tar.gz
 sudo tar -C /usr/local/ -xvzf go.tgz
 export GOROOT=/usr/local/go
 export GOPATH=$HOME/go
-```
-
-Install go dependency management tool dep
-
-```
-curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
 ```
 
 Finally, set up your Git identity and GitHub integrations.
@@ -148,7 +141,7 @@ and versioning information.
 
 Following the [GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) guidelines for Kubernetes development, set up your environment and get the latest development repository. Begin by forking both the Kubernetes and Cloud-Provider-OpenStack projects into your GitHub into your local workspace (or bringing your current fork up to date with the current state of both repositories).
 
-`make` will build, test, and package this project. This project uses [go dep](https://golang.github.io/dep/) for dependency management.
+`make` will build, test, and package this project. This project uses [go modules](https://github.com/golang/go/wiki/Modules) for dependency management since v1.17.
 
 Set up some environment variables to help download the repositories
 

